quantum/action_tapping.c

```diff
@@ -164,59 +216,31 @@ bool process_tapping(keyrecord_t *keyp) {
                  * useful for long TAPPING_TERM but may prevent fast typing.
                  */
                 // clang-format off
-#    if defined(TAPPING_TERM_PER_KEY) || (TAPPING_TERM >= 500) || defined(PERMISSIVE_HOLD) || defined(PERMISSIVE_HOLD_PER_KEY) || (defined(AUTO_SHIFT_ENABLE) && defined(RETRO_SHIFT))
                 else if (
                     (
-                        (
-                            GET_TAPPING_TERM(tapping_keycode, &tapping_key) >= 500
-#        ifdef PERMISSIVE_HOLD_PER_KEY
-                            || get_permissive_hold(tapping_keycode, &tapping_key)
-#        elif defined(PERMISSIVE_HOLD)
-                            || true
-#        endif
-                        ) && IS_RELEASED(event) && waiting_buffer_typed(event)
+                        (TAP_GET_TAPPING_TERM >= 500 || TAP_GET_PERMISSIVE_HOLD) &&
```

Comment:
```suggestion
                        TAP_GET_PERMISSIVE_HOLD &&
```
Since https://github.com/qmk/qmk_firmware/pull/15674 was merged, tapping term above 500ms no longer activate 
PERMISSIVE_HOLD-like behaviour anymore

Reply:
I rebased my branch to apply this update and also fixed the last (hopefully) Lint error.